src/nnue/features/pk.cpp

```diff
@@ -0,0 +1,50 @@
+﻿#include "pk.h"
+#include "index_list.h"
+
+//Definition of input feature PK of NNUE evaluation function
```

Comment:
Could you please explain what PK means?  Because it will be confused with KP.

Reply:
It's a union of K and P. You're right that it may be confusing with (K, P) pairs. I'll change the naming.